packages/app/src/app/graphql/types.ts

```diff
@@ -3400,10 +3400,6 @@ export type SidebarSyncedSandboxFragmentFragment = {
   __typename?: 'Sandbox';
 } & Pick<Sandbox, 'id'>;
 
-export type SidebarCollectionFragmentFragment = {
-  __typename?: 'Collection';
-} & Pick<Collection, 'id' | 'path'>;
-
```

Comment:
Removed collections from the sidebar state as they are too intertwined with the dashboard.